src/types/Profile.ts

```diff
@@ -0,0 +1,23 @@
+interface Profile {
+    id: number | string,
+    fullname: string,
+    age: number | string,
+    city: string,
+    initials: string,
+    hobbies: Hobby[],
+}
+
+interface Profile {
+    country?: string,
```

Comment:
this was added in dedicated section, to test the 'same name interface' merging behavior